code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm

```diff
@@ -0,0 +1,72 @@
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer //designed to be a very forgiving and standard abnormality.
+	name = "forsaken murderer"
+	desc = "A unhealthy looking human bound in a full body straightjacket. His neck is broken and in the middle of his forehead is a old wound that refuses to heal."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "forsakenmurdererinert"
+	icon_living = "forsakenmurdererinert"
+	icon_dead = "forsakenmurdererdead"
+	del_on_death = FALSE
+	maxHealth = 600 //originally was 270. Fragment health is 800 with a original game health of 230 so techically forsaken murderer has more health than fragment? Ill round the numbers to 600 since 270 can be rounded to 300 and doubled.
+	health = 600
+	rapid_melee = 1 //attack speed modifier. 2 is twice the normal.
+	move_to_delay = 5 //speed? lower is faster.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 6 //the lowest damage in regular attacks. Normal murderer is 2~4 so we double it.
+	melee_damage_upper = 12 // fragments Lobotomy Corp damage was 3~4 so im giving murderer a larger gap between his lower and upper damage. Unsure if i should be comparing Forsaken Murderer to Fragment of the Universe.
```

Comment:
Comparing Abnormalities of the same class is exactly what you should be doing, as most of them tend to "Swing" around the same ball-park for damage.